udf/extract_canonical_glean_app_info.sql

```diff
@@ -0,0 +1,71 @@
+/*
+
```

Comment:
Perhaps link to the historical naming doc for more context on naming.

Reply:
Are you referring to https://docs.google.com/document/d/1O_hDT8po5iEQcle62fN_eno4liVhZOhFXnKRBAoBaWk/edit?ts=5f31bb6d#heading=h.z5xwoh2241fq ?

Reply:
No, I was thinking this spreadsheet: https://docs.google.com/spreadsheets/d/18PzkzZxdpFl23__-CIO735NumYDqu7jHpqllo0sBbPA/edit#gid=0